docs/IE_DG/supported_plugins/GNA.md

```diff
@@ -189,6 +190,14 @@ executableNet.SetConfig(newConfig);
 ```
 2. Resubmit and switch back to GNA_HW expecting that the competing application has finished.
 
+> **NOTE:** This method is deprecated, since a new mode automatic QoS mode has been introduced in 2021.4.1 release of OpenVINO (see below).
```

Comment:
```suggestion
> **NOTE:** This method is deprecated since a new automatic QoS mode has been introduced in OpenVINO™  2021.4.1 release (see below).
```